### DISCUSSION TITLE
[Bug]: Editing camera group layout gets stuck on when switching groups

### DISCUSSION THREAD

**Discussion by bakedhorse** [blakeblackshear/frigate#11942] [Report a Bug] [answered]
### Describe the problem you are having

When you turn on layout editing, then switch to another group and turn it off, the original group you turned on editing will still be in edit mode without the resize border. It's fixed if you turn on and off the layout edit.

Happened in both Brave and Firefox.

### Steps to reproduce

1. Setup two different camera groups
2. Select the first group, and click on the layout edit button.
3. Switch to the second group and click on the "X" button to turn off layout edit.
4. Switch back to the first group and now you can hold down and drag any camera around while layout edit is turned off!

### Version

0.14.0-e56ce99

### Frigate config file

```yaml
Not relevant.
```

### Relevant log output

```shell
None
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Mixed

### Camera make and model

Not relevant.

### Any other information that may be helpful

Sometimes I had the layout editor turned on and the cameras cannot be moved. Only the resize border and corners are displayed but moving the cameras isn't possible. Unsure if its related, but seems to involve the same toggle for moving the cameras.

**Comment by hawkeye217** [comment, marked as answer]
This should be fixed in https://github.com/blakeblackshear/frigate/pull/11944

Once committed, you will need to clear your layouts from Settings --> General and redo them.

**Reply by bakedhorse** [reply]
Thank you very much!
